### DISCUSSION TITLE
Installing kvantum theme at system level

### DISCUSSION THREAD

**Discussion by D3vil0p3r** [tsujan/Kvantum#904] [Q&A] [answered]
According to the guide, a kvantum theme is installed inside the home folder of the user in its `.config` folder. Is it possible to install a kvantum theme at system-level instead of putting it in the home folder of each single user?

**Comment by tsujan** [comment, marked as answer]
Yes. Everything is explained in the documentation — in this case, Theme-Making.pdf → Theme Installation Paths
